src/main/java/io/opentraffic/reporter/AnonymisingProcessor.java

```diff
@@ -105,7 +101,14 @@ public void process(String key, Segment value) {
           //keep this new one
           segments.add(value);
           //put it back in the store
-          store.put(tile, segments);
+          try {
+            store.put(tile, segments);
+          }//or fail and flush to sync
+          catch (Exception e) {
+            logger.warn("Failed to store segments for tile, flushing to sync instead: " + e.getMessage());
+            store.delete(tile);
+            store(tile, segments);
```

Comment:
this is where we put to the datastore when it fails to store it in the kafka kv store